### DISCUSSION TITLE
Cant see the home page

### DISCUSSION THREAD

**Discussion by PiratePatty116** [NvChad/NvChad#3129] [Q&A] [answered]
When i type nvim in my terminal nvim gets launched but not into the home page, all i can see a file ready to be edited
![nvim](https://github.com/user-attachments/assets/2faddc32-45ba-47be-83aa-49ffeddf205e)
I am very much new to nvim so is there any command or any such trivial things that i am missing or is there anything wrong with my installation cause everything else works really well

**Comment by siduck** [comment, marked as answer]
enable nvdash on startup. read :h nvui
